hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/shell/TestOzoneShellHA.java

```diff
@@ -910,6 +912,28 @@ public void testCreateBucketWithECReplicationConfig() throws Exception {
     }
   }
 
+  @Test
+  public void testPutKeyOnBucketWithECReplicationConfig() throws Exception {
+    final String volumeName = "volume1000";
```

Comment:
The pattern in this test class is already established, but I generally prefer we use randomly generated volume, bucket and keynames where the mini-cluster is shared across multiple tests. No need to change this test for this PR, as all the existing tests have hard coded bucket / volume / key names, but just keep it in mind for the future.